Documentation/proposals/202304-silences.md

```diff
@@ -0,0 +1,108 @@
+# Silence CRD
+
+* Owners:
+  * [mcbenjemaa](https://github.com/mcbenjemaa)
+* Related Tickets:
+  * [#5452](https://github.com/prometheus-operator/prometheus-operator/issues/5452)
+  * [#2398](https://github.com/prometheus-operator/prometheus-operator/issues/2398)
+* Other docs:
+  * n/a
+
+This document describes the creation of `Silence` Custom Resource Definition that defines Alertmanager silences
+configurations in the Kubernetes way.
+
+# Why
+
+Prometheus-operator doesn't have a way to automate the management of Alertmanager silences. Users have either been using internal scripts
+or an additional operator that does the job e.g. [silence-operator](https://github.com/giantswarm/silence-operator).
+
+* Users using CI/CD jobs to manage silences have reported that this is cumbersome, since they need to either expose
```

Comment:
I would emphasize that this is even harder for multi-cluster setups as having flux and the operator running would simplify that (no need to complex CI setup or a job running in every cluster)

Reply:
I agree